pysd/builders/python/python_expressions_builder.py

```diff
@@ -656,6 +656,10 @@ def build_function_call(self, arguments: dict) -> BuildAST:
             }
 
             calls = {name: 1}
+
+        elif self.function == "elmcount":
+            final_subscripts = []
```

Comment:
For consistency and to avoid incompatibilities `final_subscripts` should be an empty dict

Reply:
Done. I added an extra commit, to make the `comp_type` of unchangeable tabbed arrays be `Unchangeable`.